diff --git a/qfluentwidgets/common/config.py b/qfluentwidgets/common/config.py
index 00c9aa9..7ef3501 100644
--- a/qfluentwidgets/common/config.py
+++ b/qfluentwidgets/common/config.py
@@ -12,7 +12,7 @@ from PySide6.QtGui import QColor
 from .exception_handler import exceptionHandler
 
 
-ALERT = "\n\033[1;33m📢 Tips:\033[0m QFluentWidgets Pro is now released. Click \033[1;96mhttps://qfluentwidgets.com/pages/pro\033[0m to learn more about it.\n"
+ALERT = ""
 
 
 class Theme(Enum):
@@ -407,11 +407,6 @@ class QConfig(QObject):
 QT_VERSION = tuple([int(v) for v in qVersion().split('.')])
 
 qconfig = QConfig()
-try:
-    print(ALERT)
-except UnicodeEncodeError:
-    print(ALERT.replace("📢", ""))
-
 
 def isDarkTheme():
     """ whether the theme is dark mode """
